lib/extensions/postgres_cdc_rls/syn_handler.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Extensions.PostgresCdcRls.SynHandler do
+  require Logger
+  alias RealtimeWeb.Endpoint
+
+  def on_process_unregistered(Extensions.PostgresCdcRls, name, _pid, _meta, reason) do
+    Logger.info("PostgresCdcRls terminated: #{inspect(name)} #{node()}")
+
+    broadcast_method =
+      if reason == :syn_conflict_resolution do
```

Comment:
`All of the callbacks, except for resolve_registry_conflict/4, are called on all the nodes of the cluster.`
- https://github.com/ostinelli/syn/blob/4b5fbc17e8008af2780af61fbf6966c15036741e/src/syn_event_handler.erl#L37

@abc3 does that mean every node's `on_process_unregistered` will be called with reason `:syn_conflict_resolution` so every node will broadcast to all the other nodes to shut down their postgres cdc process?

Reply:
`resolve_registry_conflict` will be called only once, that is why it uses `broadcast/3`